_posts/en/2021-06-08-working-in-the-open-part-2.md

```diff
@@ -0,0 +1,172 @@
+This way, you remain in control of the main version but you enable people to stop by and provide feedback on a continuous basis.
+
+### Ensure Privacy
+
+Public servants who work with citizen data must take great care regarding the privacy of that data and never share it publicly.
+Working in the open by default is not a concept that applies when working on a citizen's files and data.
+
+But it doesn't mean that even these teams can't work openly on improving their processes and sharing how they overcomed challenges.
+As public servants, we should continuously strive to improve our processes, look for new ways of solving issues, and share the results of our researches, as long as they are unclassified.
+
+### Manage Secrets
+
+In software development, certain parts of the final code may be a corporate secret, such as a private key to a computer resource, passwords, etc.
+
+These must never be included in the public source code itself but rather be managed separately.
+
+Since, software source code is nothing more than structured text that can be understood by a computer, the same concept of pulling out secrets parts may be taken with other types of documents.
+
+This means that when a small part of your work must not be shared publicly, you can exclude it and store it in a secure location instead of locking down the entire document.
+
+This way, the material that is unclassified can still be made available publicly and may be collectively worked on while the parts that are to be secured can be edited separately only by the select individuals who are authorized.
+
+### Manage Leaks
+
+In software development, we expect that there will be a security issue at some point, while we continue to strive to minimize that eventuality.
+
+By putting in place a remediation process, we ensure that even if there was to be a leak, appropriate mesures are in place to address it as quickly as possible.
+And we make sure that everyone knows how to run through the process, if it's not fully automated.
+
+In the optic of traditional text documents and presentations, reducing the amount of specific data used may help minimize the risks of leaks.
+However, someone could inadvertently type a password or personal information in a document.
+
+If that was the case, since it is not software related and automation may not be as easily possible, it would be important to have a clearly established protocol for your team members to address the data leak.
+
+### Open up at Inception
+
+Part of working in the open is accepting that we don't know everything and that others may be able to jump in and help at any time.
+By making your documents open at their inception, you put yourself in a mental state where you expect others to read and possibly participate from the get go.
```

Comment:
If changing the sentence above, this could read as 'By making your documents open at their inception, you put yourself in a mental state from the get go where you expect others to participate where and when makes sense for them'. (or something like that)